src/state/lists/hooks.ts

```diff
@@ -57,10 +58,9 @@ function listToTokenMap(list: TokenList): TokenAddressMap {
   const map = list.tokens.reduce<TokenAddressMapWriteable>((tokenMap, tokenInfo) => {
     const formattedAddress = getFormattedAddress(tokenInfo.address)
 
-    if (tokenMap[tokenInfo.chainId][formattedAddress] !== undefined) {
+    if (tokenMap[tokenInfo.chainId][formattedAddress] !== undefined || !isAddress(tokenInfo.address)) {
```

Comment:
if tokenInfo.address isnt a valid address, then formattedAddress will be "", right?

Reply:
yé

Reply:
but merged already :joy:

Reply:
hmm